anvil/src/udev.rs

```diff
@@ -716,9 +751,55 @@ fn scan_connectors(
             let allocator =
                 GbmAllocator::new(gbm.clone(), GbmBufferFlags::RENDERING | GbmBufferFlags::SCANOUT);
 
+            #[cfg(feature = "egl")]
+            let (render_feedback, scan_out_feedback) = {
+                // TODO: We should use the primary node as the main device and
+                // add the renderer as a preferred target, but as the gpu manager is
+                // late bound we can't do that here
+                let render_feedback =
+                    DmabufFeedbackBuilder::new(render_node.dev_id(), render_formats.clone())
+                        .build()
+                        .unwrap();
+
+                let planes = surface.planes().unwrap();
+                let planes_formats = surface
+                    .supported_formats(planes.primary.handle)
+                    .unwrap()
+                    .into_iter()
+                    .chain(
+                        planes
+                            .overlay
+                            .iter()
+                            .flat_map(|p| surface.supported_formats(p.handle).unwrap()),
+                    )
+                    .collect::<HashSet<_>>()
+                    .intersection(&texture_formats)
+                    .copied()
+                    .collect::<Vec<_>>();
+
+                // TODO: We should use the primary node as the main device and
+                // add the renderer as a preferred target after scan-out, but as the gpu manager is
```

Comment:
> add the renderer

`renderer`? Does that refer to the render node? I am not sure I understand this comment.

Reply:
The idea was to announce the primary gpu as the main device, a preference tranche for scan-out and a second preference tranche for the gpu on that output. But I am not sure if this will work or have any benefit.

Reply:
Ah so we would have:
- Scanout trench
- Non-scanout Trench of the optimal gpu for the multi renderer
- Non-scanout Trench of the primary gpu

Yeah, that seems reasonable. Can't we move global initialization after the udev.add_device calls to do this?

Reply:
You mean the feedback inizialization per surface? Maybe, though we need to make the feedback optional and late initialize it then. But it should be doable.

Reply:
Hm, not sure about hot-plugging though. Maybe we need to handle that in both planes.

Reply:
@Drakulix I implemented some late init for feedback which now can use the primary gpu